backend/src/utils/users.js

```diff
@@ -23,7 +23,7 @@ const validateUserPhone = (phoneNumber) => {
 
 const validateUserPin = (pin) => {
     pin = Number(pin);
```

Comment:
Isn't the pin a number after this?

Reply:
It is also generally not advisable to edit the parameters coming to your function.